packages/test-in-browser/driver.html

```diff
@@ -1,5 +1,8 @@
 
 <template name="testInBrowserBody">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@300;400;500;600;700&display=swap" rel="stylesheet">
```

Comment:
Entire Europe is getting rid of Google Fonts because of GDPR compliance. I'd strongly advise against using it.

Reply:
I did not know about that! Will make the changes ASAP